src/main/java/thestonedturtle/lootlogger/LootLoggerConfig.java

```diff
@@ -71,4 +72,13 @@ default boolean ignoreNmz()
 	{
 		return true;
 	}
-}
+
+	@ConfigItem(
+		position = 5,
+		keyName = "itemMissingAlpha",
+		name = "Missing item opacity",
+		description = "Controls the image opacity for unacquired unique items inside the uniques panel.<br/>0 is transparent and 100 is fully opaque. default value is 35"
+	)
+	@Range(max = 100)
+	default int itemMissingAlpha() { return 35; }
+}
```

Comment:
Re-add empty line at eof

I tried doing this but looks like i can't push to your PR.